tasks/47_string-expansion/string-expansion.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * stringExpansion return the expansion of that string:
+ * The numeric values represent the occurrence of each letter
+ * preceding that numeric value
+ * @param  {String} str that includes alphanumeric characters
+ * @return  {String} Returns string without numeric characters
+ */
+function stringExpansion(str) {
+  const regLetter = new RegExp(/^[A-Za-z]+$/);
+  const regNumber = new RegExp(/^[0-9]$/);
+  if (regLetter.test(str)) {
+    return str;
+  }
+  let newStr = '';
+  for (let i = 0; i < str.length; i++) {
```

Comment:
There's no need to iterate over the string. Look closely to `Regexp`. It has matching groups which could help you.

Reply:
I sound like magic character that guides you through your quest 🤣

Reply:
at the moment I’m thinking about a new solution

Reply:
I tried to fix it